account_invoice_mass_sending/tests/test_account_invoice_mass_sending.py

```diff
@@ -131,8 +129,8 @@ def test_invoice_mass_sending_3(self):
         # test one invoice to send, one with no mail and one already in progress
         self.invoices = (
             self.first_eligible_invoice
-            | self.invoice_without_mail
-            | self.invoice_in_progress
+            + self.invoice_without_mail
+            + self.invoice_in_progress
```

Comment:
I usually prefer the union over the addition because with the addition you could end up with a recordset containing twice the same id which makes no sense in Odoo.